addons/im_livechat/report/im_livechat_report_channel_views.xml

```diff
@@ -33,25 +62,35 @@
             <field name="arch" type="xml">
                 <search string="Search report">
                     <field name="channel_name"/>
-                    <field name="partner_id"/>
-                    <filter name="last_24h" string="Last 24h" domain="[('start_date','&gt;', (context_today() - datetime.timedelta(days=1)).strftime('%Y-%m-%d') )]"/>
-                    <filter name="start_date_filter" string="This Week" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(weeks=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
-                    <filter name="last_30_days" string="Last 30 Days" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(months=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
-                    <filter name="this_year" string="This Year" domain="[
-                        ('start_date', '>=', (datetime.datetime.combine(context_today() + relativedelta(years=-1,days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S')),
-                        ('start_date', '&lt;', (datetime.datetime.combine(context_today() + relativedelta(days=1,weekday=0), datetime.time(0,0,0)).to_utc()).strftime('%Y-%m-%d %H:%M:%S'))]"/>
+                    <field name="partner_id" string="Agent"/>
+                    <field name="channel_name" string="Channel"/>
+                    <field name="country_id" string="Country"/>
+                    <field name="chatbot_partner_id" string="Chatbot"/>
+                    <field name="visitor_partner_id" string="Customer"/>
+                    <field name="operator_expertise_id" string="Expertise"/>
+                    <filter name="my_session" domain="[('partner_id.user_id', '=', uid)]" string="My Sessions"/>
+                    <separator/>
+                    <filter name="escalted_session" string="Escalated" domain="[('is_without_answer', '=', 0)]"/>
```

Comment:
I don't think `is_without_answer` is the same as `escalated`. I think this should be added after https://www.odoo.com/odoo/project.task/4614101 is done.